python/lsst/ts/standardscripts/base_block_script.py

```diff
@@ -142,11 +164,22 @@ async def configure(self, config):
     async def get_obs_id(self) -> str | None:
```

Comment:
As I understand the JIRA ticket, the functionality in this method should be entirely removed. It should just return the deprecation warning and None.